code/game/objects/items/weapons/holy_weapons.dm

```diff
@@ -283,7 +283,7 @@
 
 /obj/item/weapon/nullrod/whip
 	name = "holy whip"
-	desc = "What a terrible night to be on Space Station 13."
+	desc = "What a terrible night to be on the Cyberiad!"
```

Comment:
@Fox-McCloud Its not that I'm intentionally not using the proc, the compiler throws an error about it not being constant.